src/pages/Chat/steps/CurrentInsuranceInput.tsx

```diff
@@ -0,0 +1,107 @@
+import {
+  UserResponse,
+  UserSelectInput,
+} from 'components/userInput/UserResponse'
+import { always, cond, equals, pathOr } from 'ramda'
+import * as React from 'react'
+import * as yup from 'yup'
+import { ChatContainer, CurrentInsuranceState, Insurer } from '../state'
+
+interface CurrentInsuranceInputProps {
+  appear?: boolean
+  onSubmit?: () => void
+}
+
+const insurerNames = new Map<Insurer, string>([
```

Comment:
We're missing a few here aren't we? Maybe not important for the scope of the PR though 😄

Reply:
Indeed, idk which ones should be here but imo its kind of a ui thing

Reply:
We can do the big four for now I think.

If, Folksam, Trygg-Hansa and Länsförsäkringar 🙂